rest/model/dns/answer.go

```diff
@@ -10,11 +10,18 @@ import (
 	"gopkg.in/ns1/ns1-go.v2/rest/model/data"
 )
 
+// AnswerFeed wraps the values of an Answer's "feed" attribute
+type AnswerFeed struct {
+	FeedID   string `json:"feed"`
+	SourceID string `json:"source"`
```

Comment:
Hi; I'm not sure if they can be missing, but just in case, can you make these fields omitempty? 
```suggestion
	FeedID   *string `json:"feed,omitempty"`
	SourceID *string `json:"source,omitempty"`
```

Reply:
Hi, thanks for the feedback!

As far as I know, `omitempty` is only relevant when marshaling a field, and it seems to me that these fields would never get marshaled, they are only present in the API response.

Also what would be the benefit of using pointers here? Empty strings are already invalid for this data type.

Reply:
I suppose I'm mostly worried about Terraform, who tends to write what it reads and has a bad habit of adding extra elements in arrays, both of which would result in empty strings being sent in an update; but looking at the code it will just ignore the new field, so I admit it should be fine.